client/state/cancellation-offers/selectors/is-applying-cancellation-offer.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Is there a pending request for applying a cancellation offer?
```

Comment:
They're pretty self-explaining, but parameters are missing a description. I'm mentioning it since you've added a params description to other selectors.